components/cards/ConfirmedCasesAttributesCard.vue

```diff
@@ -4,178 +4,203 @@
       <data-table
         :title="$t('陽性者の属性')"
         :title-id="'attributes-of-confirmed-cases'"
-        :chart-data="patientsTable"
-        :chart-option="{}"
+        :table-data="patientsTable"
         :date="date"
         :info="sumInfoOfPatients"
         :url="'https://catalog.data.metro.tokyo.lg.jp/dataset/t000010d0000000068'"
-        :source="$t('オープンデータを入手')"
-        :loaded="loaded"
-        :error="error"
-        :errormsg="errormsg"
+        :loaded="dataMargin >= 0"
+        :error="$fetchState.error"
         :data-length="dataLength"
         @onChangeItemsPerPage="onChangeItemsPerPage"
         @onChangePage="onChangePage"
-      />
+      >
+        <template v-slot:tableBody="{ items, headers }">
+          <tbody>
+            <tr v-for="(item, i) in items" :key="i">
+              <th scope="row" class="text-start">
+                {{ translateDate(item['公表日']) }}
+              </th>
+              <td
+                v-for="(header, j) in headers.slice(1)"
+                :key="j"
+                :class="`text-${header.align || 'start'}`"
+              >
+                <template v-if="header.type === 'date'">
+                  {{ translateDate(item[header.value]) }}
+                </template>
+                <template v-else-if="header.type === 'age'">
+                  {{ translateAge(item[header.value]) }}
+                </template>
+                <template v-else>
+                  {{ translateWord(item[header.value]) }}
+                </template>
+              </td>
+            </tr>
+          </tbody>
+        </template>
+        <template v-slot:additionalDescription>
+          <span>{{ $t('（注）') }}</span>
+          <ul>
+            <li>
+              {{ $t('「確定日」は検査により陽性であることを医師が確認した日') }}
+            </li>
+            <li>
+              {{ $t('「職業」「接触歴」「発症日」「確定日」は週１回更新') }}
```

Comment:
```suggestion
              {{ $t('「職業」「接触歴」「発症日」「確定日」は、10月1日以降の情報について週１回更新') }}
```

こちらのテキストで修正させていただきます 🙇‍♀️